include/foxy/detail/export_connect_fields.hpp

```diff
@@ -46,9 +46,7 @@ namespace detail
 // Connection = *( "," OWS ) connection-option *( OWS "," [ OWS
 // connection-option ] )
 //
-template <
-  class Fields,
-  class = std::enable_if_t<boost::beast::http::is_fields<Fields>::value>>
+template <class Fields, class = std::enable_if_t<boost::beast::http::is_fields<Fields>::value>>
```

Comment:
_Fields_ doesn't provide any specification for accessing the actual fields, the formal parameter to this function should be `basic_fields<Allocator>`, not `Fields`.